packages/terra-clinical-onset-picker/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
```

Comment:
Is this change log entry needed, I don't see any corresponding changes. Otherwise looks good.